.github/CONTRIBUTING.MD

```diff
@@ -36,4 +36,4 @@ New features are little more complex. If you come up with an idea, you should fo
 - [@tanner0101](https://github.com/tanner0101)
 - [@mcdappdev](https://github.com/mcdappdev)
```

Comment:
Can you update this to my new username? Jdmcd please! I would do it but I'm on mobile, thanks!